src/modules/Message/utils/tokens/tokenize.ts

```diff
@@ -52,20 +51,34 @@ export function identifyMentions({
 }
 
 export function identifyUrlsAndStrings(token: Token[]): Token[] {
+  const URL_REG = /(https?:\/\/|www\.)[-a-zA-Z0-9@:%._+~#=]{1,256}\.(xn--)?[a-z0-9-]{2,20}\b([-a-zA-Z0-9@:%_+[\],.~#?&/=]*[-a-zA-Z0-9@:%_+~#?&/=])*/g;
```

Comment:
We have received requests for various URL formats. (https://github.com/sendbird/sendbird-uikit-react/pull/934)
Since this involves a new URL regex different from the existing one, it would be advisable to cross-validate whether the tests for the existing URL regex are also covered or consider using a single regex for better efficiency.

Reply:
@bang9 79a5ccac968dec99defdc6581e4e1a08ab758634 Yeah it's been tested by [this test](https://github.com/sendbird/sendbird-uikit-react/blob/main/src/utils/__tests__/utils.spec.ts#L131). But we're not using `isUrl` anymore, I moved the unit test to the tokenize one.

Reply:
Thanks!